src/io/flutter/run/daemon/FlutterDaemonService.java

```diff
@@ -183,8 +183,8 @@ public FlutterApp startApp(@NotNull Project project,
                              @NotNull RunMode mode,
                              @Nullable String relativePath)
     throws ExecutionException {
-    boolean isPaused = mode.isDebug();
-    boolean isHot = mode.isDebug() ? HOT_MODE_DEFAULT : HOT_MODE_RELEASE;
+    boolean isPaused = mode.isReloadEnabled();
```

Comment:
Got it.  The variable could be better named (here and in `AppStart`).  Having done that we really want:

    startPaused = mode == RunMode.DEBUG;

👍